pkgs/by-name/ju/juce/package.nix

```diff
@@ -0,0 +1,94 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+
+  # Native build inputs
+, cmake
+, pkg-config
+, makeWrapper
+
+  # Dependencies
+, alsa-lib
+, freetype
+, curl
+, libglvnd
+, webkitgtk
+, pcre
+, darwin
+, ladspaH
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "juce";
+  version = "7.0.9";
+
+  src = fetchFromGitHub {
+    owner = "juce-framework";
+    repo = "juce";
+    rev = finalAttrs.version;
+    hash = "sha256-k8cNTPH9OgOav4dsSLqrd5PlJ1rqO0PLt6Lwmumc2Gg=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    makeWrapper
+  ];
+
+  cmakeFlags = [
+    "-DJUCE_BUILD_EXTRAS=ON"
```

Comment:
Maybe we should have an override in inputs to disable this. It can be on by default (I think this is the only way it can get cached without creating an entry in `all-packages.nix`?), but it might not always be desirable to have this.